tests/Unit/Domain/CoordinateMaps/Test_Wedge3D.cpp

```diff
@@ -548,9 +548,13 @@ SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.HalfWedge3D.Jacobian.Equiangular",
 
   for (const auto direction : Direction<3>::all_directions()) {
     const CoordinateMaps::Wedge3D map_right_half{
-        inner_radius, outer_radius, direction, sphericity, true, true};
+        inner_radius, outer_radius,
+        direction,    sphericity,
+        true,         CoordinateMaps::Wedge3D::WedgeHalves::UpperOnly};
     const CoordinateMaps::Wedge3D map_left_half{
-        inner_radius, outer_radius, direction, sphericity, true, false};
+        inner_radius, outer_radius,
+        direction,    sphericity,
+        true,         CoordinateMaps::Wedge3D::WedgeHalves::LowerOnly};
```

Comment:
serialize and deserialize the map, then call test_jacobian, etc. again on the deserialized copy